src/views/TransactionsCategories/CategpriesPage/CategoriesPage.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from "react";
+import CategoriesList from "../CategoriesList/CategoriesList";
+import categoriesService from "../../../services/api/categoriesService";
+
+const CategoriesPage = () => {
+  const [categories, setCategories] = useState([]);
+
+  const getCategories = async () => {
+    try {
+      const response = await categoriesService.get();
+      setCategories(response);
+    } catch (error) {
+      console.log(error);
```

Comment:
use throw new error or console.error because it might be that this console log wont be catched

Reply:
done